### ISSUE TITLE
No U-matic PAL signal using “Audio + Video” in passthrough

### ISSUE THREAD

**Issue by am272** [amiaopensource/vrecord#607]
I'm digitizing some U-matic PAL tapes, playing from a Sony VO-9800P. “Visual + Numerical” shows the U-matic PAL signal (for passthrough), but “Audio + Video” does now show the U-matic PAL signal (for passthrough). Tape digitizes fine though. Terminal shows these errors when I select “Audio + Video” for passthrough:

[v210 @ 0x7f9f2e847400] Application has requested 17 threads. Using a thread count greater than 16 is not recommended.
[Parsed_movie_0 @ 0x7f9f3cd33380] Channel layout is not set in output stream 1, guessed channel layout is '7.1'
[Parsed_pan_7 @ 0x7f9f1cded740] 
 
[out1 @ 0x7f9f1cdf7700] 100 buffers queued in out1, something may be wrong.

**Comment by privatezero** [comment]
Hi @ascmarsh, just want to confirm - does vrecord crash entirely when you are trying to run it with that input/filter combination?

**Comment by ascmarsh** [comment]
No, it does not crash at all. But you cannot see the video, or hear the audio on the GUI.

**Comment by privatezero** [comment]
Great - thanks for the clarification! When you have a minute, if you could upload screenshots for your configurations and for what the monitoring failure looks like on your end it might help us reproduce exactly what you are looking at on this end.
